hw_diag/utilities/miner.py

```diff
@@ -0,0 +1,77 @@
+import dbus
+import logging
+from time import sleep
+
+
+def get_public_keys():
+    """
+    get three public keys
+    PK - The public key of the miner
+    OK - The Onboarding key of the miner
+    AN - The Animal Name of the miner
+    from file "/var/data/public_keys"
+    A list of keys will be returned.
+    """
+    pk_file = None
+    while pk_file is None:
+        try:
+            pk_file = open("/var/data/public_keys").readline().split('"')[1::2]
+        except FileNotFoundError:
+            sleep(10)
+        except PermissionError:
+            raise PermissionError(
+                "/var/data/public_keys no permission to read the file"
+            )
+
+    return pk_file
+
+
+def get_miner_diagnostics():
+    # Get miner diagnostics
+    # return MC - MD - MH - MN list
+    param_list = []
+    try:
+        miner_bus = dbus.SystemBus()
+        miner_object = miner_bus.get_object('com.helium.Miner', '/')
+        miner_interface = dbus.Interface(miner_object, 'com.helium.Miner')
+        try:
+            p2pstatus = miner_interface.P2PStatus()
+            param_list = [
+                str(p2pstatus[0][1]),
+                str(p2pstatus[1][1]),
+                str(p2pstatus[3][1]),
```

Comment:
Just checking that this ordering is intentional. Is [this](https://github.com/helium/miner/blob/master/src/miner.erl#L175) the right place to look for the source data format?